dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json

```diff
@@ -32,7 +32,7 @@
     }
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}/language/authoring",
```

Comment:
Can we know the reason for this change ?

Reply:
It causes validation error. Let me revert this, and request exception.

Reply:
Standalone fix for this: https://github.com/Azure/azure-rest-api-specs/pull/20927